src/Core/Services/Implementations/LicensingService.cs

```diff
@@ -249,7 +249,7 @@ private UserLicense ReadUserLicense(User user)
             return JsonSerializer.Deserialize<UserLicense>(data);
         }
 
-        private OrganizationLicense ReadOrganizationLicense(Organization organization)
+        public OrganizationLicense ReadOrganizationLicense(Organization organization)
         {
             var filePath = $"{_globalSettings.LicenseDirectory}/organization/{organization.Id}.json";
```

Comment:
I know everyone hosts it from docker but while this is changed I really think we should use `Path.Combine` and read and deserialize asynchronously.